### PR TITLE
Clean up really long line lengths in Anderson2021

### PR DESCRIPTION
**Pull request by steven-johnson**
We don't have an explicit line length limit in Halide, but generally consider 120 to be a reasonable extent; a lot of code in Anderson2021 went waaaay over this limit, especially function/method calls. I did a semi-manual cleanup to try to clean up the worst offenders. Should be 100% cosmetic.

### CODE REVIEW

File: src/autoschedulers/anderson2021/State.h [hunk 1]
```diff
@@ -122,15 +130,28 @@ struct State {
         void add_outer_thread_loops(LoopNest *loop_nest) const;
     };
 
-    IntrusivePtr<const LoopNest> get_root_for_features(const Anderson2021Params &params, const Target &target) const;
+    IntrusivePtr<const LoopNest> get_root_for_features(const Anderson2021Params &params,
+                                                       const Target &target) const;
 
-    void set_gpu_store_site(const map<const LoopNest *, pair<const LoopNest *, int>> &parent, const LoopNest *loop, LoopNest::Sites &site) const;
+    void set_gpu_store_site(const map<const LoopNest *,
+                                      pair<const LoopNest *, int>> &parent,
```
Comment (abadams) [inline src/autoschedulers/anderson2021/State.h:137]: same here

File: src/autoschedulers/anderson2021/State.h [hunk 2]
```diff
@@ -86,11 +90,15 @@ struct State {
     uint64_t structural_hash(int depth) const;
 
     // Compute the parent and depth of every loop nest node
-    void compute_loop_nest_parents(map<const LoopNest *, pair<const LoopNest *, int>> &p,
-                                   const LoopNest *here, int depth) const;
+    void compute_loop_nest_parents(map<const LoopNest *,
```
[the remark this answers is not in the document]
Comment (steven-johnson) [reply src/autoschedulers/anderson2021/State.h:93]: Oh, good catch, I misread the comma in manual formatiing, willfix

File: src/autoschedulers/anderson2021/State.h (hunk 1 above)
Comment (steven-johnson) [reply src/autoschedulers/anderson2021/State.h:137]: Oh, good catch, I misread the comma in manual formatiing, willfix

**Comment by steven-johnson** [comment]
Fixed the weirdness by adding a type alias for the funky map type, since it's used in several places

**Comment by steven-johnson** [comment]
Failures unrelated, PTAL

**Review by abadams — APPROVED** [review]

**Pull request merged**